swmm-toolkit/setup.py

```diff
@@ -116,10 +116,10 @@ def exclude_files(cmake_manifest):
         "Operating System :: POSIX :: Linux",
         "Operating System :: MacOS",
         "License :: CC0 1.0 Universal (CC0 1.0) Public Domain Dedication",
-        "Programming Language :: Python :: 3.6",
         "Programming Language :: Python :: 3.7",
         "Programming Language :: Python :: 3.8",
         "Programming Language :: Python :: 3.9",
+        "Programming Language :: Python :: 3.10",
         "Programming Language :: C",
         "Development Status :: 4 - Beta",
```

Comment:
Is development still in BETA? Can we remove this?

Reply:
Yeah, let's do it. I remove it and update it to production/stable.